usr/share/rear/prep/CDM/default/400_prep_cdm.sh

```diff
@@ -0,0 +1,7 @@
+#
+# prepare stuff for CDM
+#
+
+COPY_AS_IS=( "${COPY_AS_IS[@]}" "${COPY_AS_IS_CDM[@]}" )
+COPY_AS_IS_EXCLUDE=( "${COPY_AS_IS_EXCLUDE[@]}" "${COPY_AS_IS_EXCLUDE_CDM[@]}" )
+PROGS=( "${PROGS[@]}" "${PROGS_CDM[@]}" fmt )
```

Comment:
Only as you like:

At ReaR upstream we think that
```
ARRAY+=( additional elements )
```
is simpler and more fail safe than
```
ARRAY=( "${ARRAY[@]}" additional elements )
```
see
https://github.com/rear/rear/issues/2220#issuecomment-528343007
and subsequent comments there.

Reply:
Merged as is.